recipes/xbyak/all/conanfile.py

```diff
@@ -32,4 +32,3 @@ def package(self):
     def package_info(self):
         self.cpp_info.names["cmake_find_package"] = "xbyak"
         self.cpp_info.names["cmake_find_package_multi"] = "xbyak"
-        self.cpp_info.set_property("cmake_file_name", "xbyak")
```

Comment:
can you explain why this is removed ?

Reply:
oh yes it's `cmake_file_name`, so it doesn't matter. I close this PR.